client/src/components/common/index.js

```diff
@@ -0,0 +1,5 @@
+import FilterResult from './FilterResult';
+import Loader from './Loader';
+import Card from './Card';
+
+export { FilterResult, Loader, Card };
```

Comment:
I think eslint didn't work in yours , 
```suggestion
export { 
FilterResult, 
Loader, 
Card };
```